framework_tvb/tvb/interfaces/web/controllers/flow_controller.py

```diff
@@ -696,7 +697,7 @@ def get_adapter_template(self, project_id, algorithm_id, is_upload=False, back_p
     def reloadoperation(self, operation_id, **_):
         """Redirect to Operation Input selection page, 
         with input data already selected."""
-        operation = self.flow_service.load_operation(operation_id)
+        operation = self.operation_services.load_operation(operation_id)
```

Comment:
Maybe call the static load_operation directly from the OperationService class? In this way we will keep the consistency in the flow controller.